packages/flutter_tools/lib/src/test/runner.dart

```diff
@@ -29,8 +30,14 @@ Future<int> runTests(
     bool ipv6: false,
     bool machine: false,
     bool previewDart2: false,
+    bool trackWidgetCreation: false,
     TestWatcher watcher,
     }) async {
+  if (trackWidgetCreation && !previewDart2) {
+    throw new UsageException(
+        '--track-widget-creation is valid only when --preview-dart-2 is specified.', null);
```

Comment:
you can make this a one-liner if you prefer. Or alternatively:
```dart
  throw new UsageException(
    '--track-widget-creation is valid only when --preview-dart-2 is specified.',
     null,
  );
￼```

Reply:
done